libs/sdk-ui-dashboard/src/model/events/insight.ts

```diff
@@ -658,15 +659,15 @@ export interface DashboardInsightWidgetExportResolved extends IDashboardEvent {
  */
 export function insightWidgetExportResolved(
     ctx: DashboardContext,
-    resultUri: string,
+    result: IExportResult,
     correlationId?: string,
 ): DashboardInsightWidgetExportResolved {
     return {
         type: "GDC.DASH/EVT.INSIGHT_WIDGET.EXPORT_RESOLVED",
         ctx,
         correlationId,
         payload: {
-            resultUri,
+            result,
```

Comment:
Can this be changed if this is beta? Possibly some plugins would require update?
Maybe I should change the payload to:

````
payload: {
	resultUri,
	result,
},
````

This way it should be compatible. There's just the issue with `objectURL` that will no longer be valid once download finishes.